src/sql/workbench/browser/modal/media/modal.css

```diff
@@ -263,43 +334,58 @@
 
 .hc-black .dialog-message.warning .close-message-icon,
 .close-message-icon {
-	background: url('close_inverse.svg') center center no-repeat;
+	background: url("close_inverse.svg") center center no-repeat;
 }
 
 .dialog-message.warning .close-message-icon {
-	background: url('close.svg') center center no-repeat;
+	background: url("close.svg") center center no-repeat;
 }
 
 .hc-black .dialog-message.warning .copy-message-icon,
 .copy-message-icon {
-	background: url('copy_inverse.svg') center center no-repeat;
+	background: url("copy_inverse.svg") center center no-repeat;
 }
 
 .dialog-message.warning .copy-message-icon {
-	background: url('copy.svg') center center no-repeat;
+	background: url("copy.svg") center center no-repeat;
 }
 
 .hc-black .dialog-message.warning .message-details-icon,
 .message-details-icon {
-	background: url('show_details_inverse.svg') center center no-repeat;
+	background: url("show_details_inverse.svg") center center no-repeat;
 }
 
 .dialog-message.warning .message-details-icon {
-	background: url('show_details.svg') center center no-repeat;
+	background: url("show_details.svg") center center no-repeat;
 }
 
-.dialog-message.info .dialog-message-icon  {
-	background: url('info_notification_inverse.svg') center center no-repeat;
+.dialog-message.info .dialog-message-icon {
+	background: url("info_notification_inverse.svg") center center no-repeat;
 }
 
 .hc-black .dialog-message.warning .dialog-message-icon {
-	background: url('warning_notification_inverse.svg') center center no-repeat;
+	background: url("warning_notification_inverse.svg") center center no-repeat;
```

Comment:
just curious any reason you are changing the single quote to double quote?

Reply:
After committing changes, I was warned by the linter that the CSS needed to be formatted. I then just formatted the file and checked it in.

Reply:
Replaced " with '